### PR TITLE
Cleaned postgres pkg

### PR DESCRIPTION
**Pull request by bokimilinkovic**
https://trello.com/c/NePM2Rxp/59-clean-up-postgres-pkg

### CODE REVIEW

File: repository/postgres/postgres.go [hunk 1]
```diff
@@ -16,25 +16,35 @@ type PostgresConfig struct{
 }
 
 
-type dbStore struct {
+type Store struct {
+	Db db
+}
+type db struct{
```
Comment (sfqi) [inline repository/postgres/postgres.go]: type db *gorm.DB

File: repository/postgres/postgres.go [hunk 2]
```diff
@@ -16,25 +16,35 @@ type PostgresConfig struct{
 }
 
 
-type dbStore struct {
+type Store struct {
+	Db db
```
Comment (sfqi) [inline repository/postgres/postgres.go]: Also unexport this

File: repository/postgres/postgres.go [hunk 3]
```diff
@@ -17,34 +17,29 @@ type PostgresConfig struct{
...
+	db *gorm.DB
 }
 
 func NewStore()*Store{
-	return &Store{
-		Db:db{
-			DB:&gorm.DB{},
-		},
-	}
+	return &Store{}
```
Comment (sfqi) [inline repository/postgres/postgres.go]: How about `NewStore(db *gorm.DB) *Store`?

Then you could actually use it in `Open`.

Comment (bokimilinkovic) [reply repository/postgres/postgres.go]: good idea

**Pull request merged**